iris_ugrid/ugrid_cf_reader.py

```diff
@@ -179,19 +192,27 @@ def __init__(self, filename, *args, **kwargs):
 
     def cube_completion_adjust(self, cube):
         """
-        Cube post-processing method to add details of the mesh to any newly
-        created cubes which have a mesh dimension.
+        Cube post-processing method convert newly created cubes which have a
+        a mesh dimension into :class:`UCubes`s.
 
         Called by a 'cube post-modify hook' in
         :func:`iris.fileformats.netcdf.load_cubes`.
 
-        Adds the ".ugrid" property to cubes created by the CF reader, which
-        links the cube mesh dimension to a specific mesh and element-type (aka
-        "mesh_location").
+        Constructs a :class:`CubeUgrid` referencing the appropriate file mesh,
+        and makes a new `UCube` of which this is the '.ugrid' property.
+
+        Args:
+
+        * cube (:class:`iris.cube.Cube`):
+            The cube to be post-processed
+
+        Returns:
+            :class:`iris_ugrid.ucube.UCube`
```

Comment:
... or None. 
It may be worth mentioning that?

Reply:
33b8fc76515c795057ed4315bac4a411eb051d49